0.0.6

```diff
@@ -71,17 +71,17 @@
         {
             "sObjectName": "PersonAccount",
             "instructionTypeName": "command__Client",
-            "description": "Add Type field to AC Task Layout",
+            "description": "Add Gift related list to Client Layout",
```

Comment:
Gifts AND files. Being picky since you'll have to update the below description anyways.